.github/workflows/eval-lib-tests.yml

```diff
@@ -0,0 +1,27 @@
+name: "Building Nixpkgs lib-tests"
+
+permissions:
+  contents: read
+
+on:
+  pull_request_target:
+    branches-ignore:
+      - 'staging'
+    paths:
+      - 'lib/**'
+jobs:
+  nixpkgs:
+    name: nixpkgs-lib-tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          # pull_request_target checks out the base branch by default
+          ref: refs/pull/${{ github.event.pull_request.number }}/merge
+      - uses: cachix/install-nix-action@08dcb3a5e62fa31e2da3d490afc4176ef55ecd72 # v30
+        with:
+          # explicitly enable sandbox
+          extra_nix_config: sandbox = true
+      - name: Building Nixpkgs lib-tests
+        run: |
+           nix-build --arg pkgs "import ./. {}" ./lib/tests/release.nix
```

Comment:
If you use the pinned Nixpkgs version from the ci directory for `pkgs`, there's no need to exclude certain branches, because no dependencies should come from the Nixpkgs being tested.

Reply:
Something like this  `nix-build --arg pkgs  "(import ./ci/. {}).pkgs"`

Reply:
Done, thanks.